.github/workflows/pr_master_test.yml

```diff
@@ -202,6 +208,7 @@ jobs:
             || echo "WARNING: Xpress Community Edition is not available"
         python -c 'import sys; print("::set-env name=PYTHON_EXE::%s" \
             % (sys.executable,))'
+        echo "::set-env name=NOSETESTS::"`which nosetests`
```

Comment:
What's the purpose of this?

Reply:
It's the `nosetests` equivalent of `$PYOMO_EXE`: just in case paths to the configured python break, we can "remember" the path to nosetests (for use in the MPI tests command line)

Reply:
I like it. Future-proofing (or, OSX proofing).